srcpkgs/scheme48/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'scheme48'
+pkgname=scheme48
+version=1.9
+revision=2
+archs="x86_64 x86_64-musl"
```

Comment:
The problem is cross compiling, not tying to architecture. Use `nocross="failed to compile test program"` instead.